app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt

```diff
@@ -121,7 +124,6 @@ class MeetingRequestService : FirebaseMessagingService() {
             .setContentText(message)
             .setPriority(NotificationCompat.PRIORITY_DEFAULT)
             .setAutoCancel(true)
-
     notificationManager.notify(NOTIFICATION_ID, notificationBuilder.build())
```

Comment:
NOTIFICATION_ID is hardcoded to 0, shouldn't notifications have unique ids to ensure they don't overwrite each other?

Reply:
Okay we will see!